.clang-format

```diff
@@ -22,3 +22,33 @@ PointerAlignment: Left
 Standard: c++14
 TabWidth: 2
 UseTab: Never
+IncludeBlocks: Regroup
+IncludeCategories:
+  - Regex:           '^<pcl/'
+    Priority:        1
+  - Regex:           '^<gtest/'
+    Priority:        2
+  - Regex:           '^<boost/'
+    Priority:        3
+  - Regex:           '^<(unsupported/)?Eigen/'
+    Priority:        4
+  - Regex:           '^<flann/'
+    Priority:        5
+  - Regex:           '^<(Q|q)\w*/'
+    Priority:        6
+  - Regex:           '^<vtk/'
+    Priority:        7
+  - Regex:           '^<librealsense2/'
+    Priority:        8
+  - Regex:           '^<(ros|message_filters)/'
+    Priority:        9
+  - Regex:           '^<opencv(2)?/'
+    Priority:        10
+  - Regex:           '^<tide/'
+    Priority:        11
+  - Regex:           '^<thrust/'
+    Priority:        12
+  - Regex:           '^<(OpenGL|(GL(UT)?)/'
+    Priority:        13
+  - Regex:           '.*'
+    Priority:        14
```

Comment:
Would it make sense to add`^<[a-z]*>` (cpp headers, priority 15) as different group?

Reply:
Where is the difference to the last rule? Due to the directory structure includes with `<>` are the only correct one, as we have never relative files to current file.

Reply:
The regex is very different. Having only a-z inside angle brackets is different from any sequence of characters.

Reply:
Ahh you want unknown 3rd-Party components filtered. Good idea 👍